### ISSUE TITLE
Link to new Persistent Object Caching documentation from Autoloaded Options health check

### ISSUE THREAD

**Issue by mxbclang** [WordPress/performance#449]
As discussed in #234 and being reviewed in #448, once the new Persistent Object Caching section is ready and added, add a link to it in the [autoloaded options](https://wordpress.org/support/article/optimization/#autoloaded-options) section:

Add a third one-sentence paragraph: "If you use a Persistent Object Cache, options (whether autoloaded or not) load faster and more efficiently." and link "Persistent Object Cache" to the new guide section.

**Comment by felixarntz** [comment]
This is a duplicate of #407, which was fixed about a year ago :)
